src/scripts/Controller.js

```diff
@@ -1324,15 +1363,27 @@ export default class SitemapController {
 	}
 
 	addSelector() {
-		const parentSelectorId = this.state.currentParentSelectorId;
 		const sitemap = this.state.currentSitemap;
+		const parentSelectorId =
+			sitemap.extensionVersion > 0
+				? this.state.currentParentSelectorId.uuid
+				: this.state.currentParentSelectorId.id;
 
 		const selector = SelectorList.createSelector({
 			parentSelectors: [parentSelectorId],
 			type: 'SelectorText',
 			multiple: false,
+			uuid: String(
+				Number(
+					this.state.currentSitemap.selectors
+						.map(function (el) {
+							return el.uuid;
+						})
+						.max()
+				) + 1
+			),
```

Comment:
- prefer arrow functions: `.map(el => el.uuid)` or even `.map(({ uuid }) => uuid)`
- maybe we can use integer uuids instead of parsing them from string to number and back?

Reply:
```suggestion
			uuid: String(
				Math.max(0, ...this.state.currentSitemap.selectors.map(({ uuid }) => uuid)) + 1
			),
```